.travis.yml

```
@@ -21,13 +21,15 @@ env:
      - DOCKER_USERNAME="fairspace"
      - GITHUB_USERNAME="fairspace-ci"
      - GCP_PROJECT="fairspace-207108"
-     - SNAPSHOT_CONTEXT="gke_fairspace-207108_europe-west1-b_fairspacecicluster"
-     - SNAPSHOT_CLUSTER_NAME="fairspacecicluster"
-     - SNAPSHOT_CLUSTER_ZONE="europe-west1-b"
-     - RELEASE_CONTEXT="gke_fairspace-207108_europe-west1-b_fairspacecicluster"
-     - RELEASE_CLUSTER_NAME="fairspacecicluster"
-     - RELEASE_CLUSTER_ZONE="europe-west1-b"
      - GIT_TAG_FILES_OVERRIDE="charts/fairspace/Chart.yaml charts/fairspace/values.yaml projects/"
+     - SHOULD_DEPLOY=0
```

Maybe use 'false' for better readability?
```suggestion
     - SHOULD_DEPLOY=false
```

---

I wanted to be consistent with `SHOULD_RELEASE` flag from [fairspace-build-scripts repo](https://github.com/thehyve/fairspace-build-scripts/blob/master/setup_env.sh#L13), but it looks like that one is either `1` or `undefined`. I'll make the deploy one `true` \ `false` then.

---

Weird language, this 'bash'... :)

---

So, `[[ "$CONDITION" ]]` evaluates to false when `"$CONDITION"` is an empty string.

---

yeah, this is so intuitive... :sweat_smile: